Program.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Diagnostics.Eventing.Reader;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace TasksProgCs
+{
+    internal class Program
+    {
+        static void Main(string[] args)
+        {
+            Console.WriteLine("Введите значение X");
+            double x = Double.Parse(Console.ReadLine());
+            Console.WriteLine("f(x)=" + CalculateFunction(x));
+        }
+        static double CalculateFunction(double x)
+        {
+            if (x > 1) { return Math.Log(x); }
+
+            else if (x > 0 && x <= 1)
+            { return -Math.Log(x); }
+            else { return 0; }; 
```

Comment:
Скобки лишние (тело клаузы составляет один оператор) . Повторное сравнение с 1 лишнее. Последняя точка с запятой лишняя.
```suggestion
            if (x > 1)
                 return Math.Log(x);
            else if (x > 0)
                 return -Math.Log(x);
            else
                 return 0;
```